pkg/engine/task/task_kudo_operator.go

```diff
@@ -11,9 +30,165 @@ type KudoOperatorTask struct {
 	InstanceName    string
 	AppVersion      string
 	OperatorVersion string
+	ParameterFile   string
 }
 
 // Run method for the KudoOperatorTask. Not yet implemented
 func (dt KudoOperatorTask) Run(ctx Context) (bool, error) {
-	return false, errors.New("kudo-operator task is not yet implemented. Stay tuned though ;)")
+
+	// 0. - A few prerequisites -
+	// Note: ctx.Meta has Meta.OperatorName and Meta.OperatorVersion fields but these are of the **parent instance**
+	// However, since we don't support multiple namespaces yet, we can use the Meta.InstanceNamespace for the namespace
+	namespace := ctx.Meta.InstanceNamespace
+	operatorName := dt.Package
```

Comment:
That a very good point (which I forgot to document). It is called `Pacakge` because that's what [we call it](https://github.com/kudobuilder/kudo/blob/master/pkg/kudoctl/cmd/install.go#L14) on the CLI side. Since `O/OV` resources are created on the client-side the package must be resolved there too. Here, the same resolution rules apply e.g. local path has a priority before remote repo. This might come handy when debugging/developing operators, where a child-operator can (temporarily) be installed from a local folder. However, once we're on the server-side, this has to be an operator name (overridden by the CLI?).